shotover/src/runner.rs

```diff
@@ -129,6 +138,11 @@ impl Shotover {
             tracing::info!(
                 "Hot reloading is ENABLED - shotover will support hot reload operations"
             );
+        } else if params.hotreload_from_socket.is_some() {
```

Comment:
This should be written as `if let Some(socket_path) = params.hotreload_from_socket {`, then we can use `socket_path` in the info log below and avoid needing to call `.unwrap()`.

It is always preferable to use `if let` instead of `is_some()` + `unwrap()` since its easier to tell its correct at a glance.